src/index.html

```diff
@@ -2,13 +2,115 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="header-logo-container">
+          <a href="#">
+            <img
+              class="header-logo"
+              src="./images/logo.png"
+              alt="Moyo-lodo"
+            >
+          </a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="list_item">
+              <a
+                class="nav-link is-active"
+                href="http://"
+              >
+                APPLE
```

Comment:
```suggestion
                Apple
```
используй text-transform: uppercase